assets/src/scripts/store/index.js

```diff
@@ -72,11 +72,16 @@ export const Actions = {
                 series => {
                     const collection = normalize(series, requestKey);
 
+                    // get the lat/lon of the site
+                    const location = collection.sourceInfo ? collection.sourceInfo[siteno].geoLocation.geogLocation : {};
+                    const latitude = location.latitude || null;
+                    const longitude = location.longitude || null;
+
                     // Get the start/end times of this request's range.
                     const notes = collection.queryInfo[requestKey].notes;
                     const endTime = notes.requestDT;
                     const startTime = calcStartTime('P7D', endTime, 'local');
-
+                    dispatch(Actions.retrieveLocationTimeZone(latitude, longitude));
```

Comment:
Should this be conditional to latitude/longitude being non-null?

Reply:
It doesn't have to be. Things will work fine it doesn't without a condition, but it'd  be cleaner with it.
I'll add it.